<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Warehouse_dialog</class>
 <widget class="QDialog" name="Warehouse_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>615</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>-10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>-10</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="capacity">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>-10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="stock">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>-10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="nail_l">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="yonje_l">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bil_l">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="milk_l">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pashm_l">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="egg_l">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nail_nl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="yonje_nl">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="bil_nl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="pashm_nl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="milk_nl">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="egg_nl">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;

</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upgrade">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="lev">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 85, 0);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>530</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 85, 0);</string>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
